### PR TITLE
Clarify what RUN_ONCE returns

### PR DESCRIPTION
**Pull request by levitte**
##### Checklist

- [x] CLA is signed

### CODE REVIEW

File: include/internal/thread_once.h [hunk 1]
```diff
@@ -36,7 +36,8 @@
  * @init: function name that was previously given to DEFINE_RUN_ONCE,
  *        DEFINE_RUN_ONCE_STATIC or DECLARE_RUN_ONCE.
  *
- * The return value is 1 on success or 0 in case of error.
+ * The return value is 0 in case of error.  Otherwise, it is whatever
+ * the init function returns on the first call.
```
Comment (kaduk) [inline include/internal/thread_once.h]: "on the first call", well, there is only one call, right?

**Review by kaduk — APPROVED** [review]
Comment inline notwithstanding, LGTM

File: include/internal/thread_once.h (hunk 1 above)
Comment (levitte) [inline include/internal/thread_once.h]: The first call of RUN_ONCE with the same parameters...

**Comment by levitte** [comment]
@kaduk, I simplified the comment.

**Comment by levitte** [comment]
After much deliberation in #2222, it was clear that my initial change was contrary to the intention.  I've changed the clarification to match.

**Review by mattcaswell — APPROVED** [review]

**Comment by levitte** [comment]
I changed the commit message to fit.

Merged.  master: 37b8f1e21c56996644afa38816f575b05eb483ae, 1.1.0: ba2efdc9ed75659a75d8b8c3cd4ae5bd3c181c65

**Pull request closed without merging**